src/components/DateTimePickerNative/DateTimePickerNative.vue

```diff
@@ -0,0 +1,186 @@
+<!--
+  - @copyright Copyright (c) 2022 Julia Kirschenheuter <julia.kirschenheuter@nextcloud.com>
+  -
+  - @author Julia Kirschenheuter <julia.kirschenheuter@nextcloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<docs>
+> We're wrapping the awesome native browser datepicker as an input with some type of date e.g. https://html.spec.whatwg.org/multipage/input.html#local-date-and-time-state-(type=datetime-local)
+> Please check there for all the available options: https://html.spec.whatwg.org/multipage/input.html#
+> All available types are: 'date', 'datetime-local', 'month', 'time' and 'week'.
+
+### Example: datetime-local
+```vue
+<template>
+	<span>
+		<DateTimePickerNative
+			v-model="value"
+			:label="label"
+			type="datetime-local" />
+	</span>
+</template>
+<script>
+	export default {
+		data() {
+			return {
+				value: new Date(),
+				label: 'please select a new date'
+			}
+		},
+	}
+</script>
+```
+
+</docs>
+
+<template>
+	<div class="native-datetime-picker">
+		<label :class="{ 'hidden-visually': noVisibleLabel }"
+			:for="`native-datetime-picker_${label}`">{{ label }}</label>
```

Comment:
```suggestion
```

IMO, the label should not be part of the component. We should be able to pass down an id prop to the input and then just add a label externally (if required).

Reply:
i agree with availability of ``` id ```. But i still think that ``` label ``` is there in any case (accessibility issue). But i can change the default value to indivisible.

Reply:
Let's make it consistent with other inputs. Do they have a label internally? Then we leave it like it is. If they have none either we move this responsibility to the users of the component.

Reply:
Thank for your thoughts! IMO i would like to leave ```label``` here, because in this way it has right semantics and is accessible. Input can't be without label. If the user of component will not provide a label - then will will have a several amount of accessibility issues again.

Reply:
As per https://github.com/nextcloud/nextcloud-vue/pull/3063#discussion_r959590561: https://github.com/nextcloud/nextcloud-vue/search?q=%3Clabel.

So we leave the label for consistency.